src/HM6/Bobik.java

```diff
@@ -0,0 +1,34 @@
+package HM6;
+
+public class Bobik extends Animal{
+    private int _lenghtSwim;
+    private int maxLenght;
+
+    Bobik(){
```

Comment:
следует использовать параметризированный конструктор, чтобы через super(...) инициализировать поля родительского класса:
```suggestion
    public Bobik(int maxLength, int maxJump, int maxSwim) {
        super(maxLength, maxJump, maxSwim);
```
соответствующий конструктор должен быть в родительском классе, как и все поля для инициализации